packages/serverless-cdk-plugin/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@swarmion/serverless-cdk-plugin",
+  "private": true,
```

Comment:
This needs to be put to `true` for publication

Reply:
The key should not exist for public package